src/cli-options.ts

```diff
@@ -194,14 +197,16 @@ export const getOptions = (): Result<Options> => {
     };
   }
 
+  const agent = getNpmUserAgent();
```

Comment:
Is there any warning for an unknown agent?

Reply:
There is!
https://github.com/google/wireit/blob/3ed29fc16c642ecec85546284869156c44f0d803/src/cli-options.ts#L277-L281

Minor nit relating to that, the return type of the `getNpmUserAgent` can now use the `Agent` type.
https://github.com/google/wireit/blob/3ed29fc16c642ecec85546284869156c44f0d803/src/cli-options.ts#L265

Reply:
Done!